python/nav/web/templates/custom_crispy_templates/horizontal_checkbox.html

```diff
@@ -1,9 +1,9 @@
-{% load crispy_forms_field %}
+{% load forms %}
```

Comment:
You're not actually using any template tags from the `forms` template library in this template, so I don't see the point of loading it...